Load data from local files

geometer opened the pull request:
Misha: please review this code

Commit:
enabled loading data from CEL files located in the same directory as IDF/SDRF files (http://bar.ebi.ac.uk:8080/trac/ticket/2993)

Commit:
enabled loading experiment data from the files in same directory where IDF/SDRF are located

gxa commented:
Added a comment on the commit - would rather get rid of the boolean variable, and extract local processing into a method.

alf239 commented:
@gxa @ostolop try choosing one identity :)

The comments you add to a commit in the pull request are automatically added to the pull request itself, there's no real need to comment twice (other than replies counter in the pull requests list)

ostolop commented:
:) Woops - wasn't sure about it.

Commit:
correct error message after failure in R code

Conflicts:

	atlas-loader/src/main/java/uk/ac/ebi/gxa/loader/steps/ArrayDataStep.java
